main/src/library/Array.flix

```diff
@@ -47,6 +47,13 @@ instance Iterable[Array[a, r]] {
         checked_ecast(Array.iterator(rc, a))
 }
 
+instance ForEach[Array[a, r]] {
+    type Elm = a
+    type Aef = r
+    pub def forEach(f: a -> Unit \ ef, a: Array[a, r]): Unit \ ef + r =
+        Array.forEach(f, a)
```

Comment:
I think you should just put the call on the same line as the signature. You did that elsewhere I saw.

Reply:
So just do that everywhere.

Reply:
I was matching the existing style of each file but i can do same line everywhere if that's preferred